### DISCUSSION TITLE
Wiki Js port not working

### DISCUSSION THREAD

**Discussion by Logan870000** [requarks/wiki#5621] [Help / Questions]
hello, 

ive just installed wiki js on debian 11 VM, in local, with ipserver:port its working, but when i want to do the same on my computer with ipserver:port, its not working, i use port 5555, i have maybe miss one thing on the configuration, but i dont know what.... if someone can help me. 

thanks :)

**Comment by cdn0x12** [comment]
Check the firewall settings of your VM or set up a reversed proxy to forward the port to other accessible ports.
